README.md

```diff
@@ -1,11 +1,11 @@
-[![License: GPL v2+](https://img.shields.io/badge/License-GPL%20v2+-blue.svg)](LICENSE.md)
+[![License: GPL-2.0-or-later](https://img.shields.io/badge/License-GPL%20v2+-blue.svg)](LICENSE.md)
```

Comment:
Why is the previous badge not ok? Is gplv2+ also works.

Reply:
The name itself does not matter, like before was also OK, thought only to have equal to them on `addon.xml`.

This there is only a fallback if the image not available.